### DISCUSSION TITLE
Line break in info field - config.toml

### DISCUSSION THREAD

**Discussion by magalios** [luizdepra/hugo-coder#736] [Q&A] [answered]
Hi, 
great theme, really enjoy it. i try to figure out putting a line break in the info field of the config file. Any ideas? 
Thanks

**Comment by luizdepra** [comment, marked as answer]
Hey!

The `info` field is a list, if you put two sentences, they will stack on top of each other. See [the demo here](https://hugo-coder.netlify.app/).

```
info = ["First line", "Second line"]
```

But, if you need an empty line, you just could add an empty string in the list. Something like this:

```
info = ["First line", "", "Second line"]
```
